server/ladokApi.js

```diff
@@ -5,6 +5,16 @@ const { server: serverConfig } = require('./configuration')
 
 const client = createApiClient(serverConfig.ladokMellanlagerApi)
 
+function formatExaminationTitles(examinationModules, lang) {
+  const liStrs = examinationModules.map(
+    m =>
+      `<li>${m.kod} - ${m.benamning}, ${m.omfattning.formattedWithUnit}, ${lang === 'sv' ? 'Betygsskala' : 'Grading scale'}: ${m.betygsskala.code}</li>`
+  )
+  const titles = examinationModules.map(m => `<h4>${m.kod} - ${m.benamning}, ${m.omfattning.formattedWithUnit}</h4>`)
+  const formatted = { liStrs: liStrs.join(), titles: titles.join() }
```

Comment:
Default separator for `join()` is `,` so this will add an unwanted `,` between each row. So change to an empty string as arg. 
```suggestion
  const formatted = { liStrs: liStrs.join(''), titles: titles.join('') }
```